### PR TITLE
Ks/#2704 subscription providers

### PR DESCRIPTION
**Pull request by KSchopmeyer**
Adds pywbem_mock providers for the indication subscription classes.

NOTES:
1. Better way to define install.- Right now I added a new method to pywbem_mock/_wbemconnection_mock  I think it would be better if each provider carried its own initialization, a common call) **DISCUSSION**. Note that the install for the namespace provider and these providers is effectively the same code.  It is just that we are intalling 3 providers in this one from a single file. We agreed that this is an issue to be resolved separately.  See comments below for more details.  This method is marked experimental.

### CODE REVIEW

File: pywbem_mock/_wbemconnection_mock.py [hunk 1]
```diff
@@ -497,6 +500,67 @@ def install_namespace_provider(self, interop_namespace,
                                schema_pragma_files=schema_pragma_file,
                                verbose=verbose)
 
+    def install_subscription_providers(self, interop_namespace,
+                                       schema_pragma_file=None,
+                                       verbose=None):
+        """
```
Comment (andy-maier) [inline pywbem_mock/_wbemconnection_mock.py:506]: Mark it experimental, since we think it will go away again.

**Review by andy-maier — CHANGES_REQUESTED** [review]
Approved except for the one comment.

File: pywbem_mock/_wbemconnection_mock.py (hunk 1 above)
Comment (KSchopmeyer) [reply pywbem_mock/_wbemconnection_mock.py:506]: Marked as experimental inn the code

**Comment by KSchopmeyer** [comment]
**DISCUSSION**

The python 2.7 failure is"

Collecting GitPython==2.1.1 (from -c minimum-constraints.txt (line 182))
  Could not find a version that satisfies the requirement GitPython==2.1.1 (from -c minimum-constraints.txt (line 182)) (from versions: )
No matching distribution found for GitPython==2.1.1 (from -c minimum-constraints.txt (line 182))

**Comment by KSchopmeyer** [comment]
Collecting GitPython==2.1.1 (from -c minimum-constraints.txt (line 182))
  Could not find a version that satisfies the requirement GitPython==2.1.1 (from -c minimum-constraints.txt (line 182)) (from versions: )
No matching distribution found for GitPython==2.1.1 (from -c minimum-constraints.txt (line 182))
You are using pip version 19.2.3, however version 21.1.3 is available.

**Comment by andy-maier** [comment]
The GitPython issue seems to have resolved itself. See also issue #2725 .

**Review by andy-maier — APPROVED** [review]

**Pull request merged**